src/core/qfieldcloud/qfieldcloudproject.cpp

```diff
@@ -739,9 +748,9 @@ void QFieldCloudProject::download()
     mLastExportId = packageId;
     mLastExportedAt = packagedAt;
 
-    if ( !localizedDatasets.isEmpty() && !mLocalizedDatasetsProjectId.isEmpty() )
+    if ( !localizedDatasetsFileNames.isEmpty() && !mSharedDatasetsProjectId.isEmpty() )
```

Comment:
Is there a reason why you kept a few localized in here? Eg this, the network reply variable name, etc